base/abstractarray.jl

```diff
@@ -2396,8 +2396,9 @@ pushfirst!(A, a, b, c...) = pushfirst!(pushfirst!(A, c...), a, b)
 
 ## hashing AbstractArray ##
 
+const hashabstrastarray_seed = UInt === UInt64 ? 0x7e2d6fb6448beb77 : 0xd4514ce5
 function hash(A::AbstractArray, h::UInt)
-    h = hash(AbstractArray, h)
+    h += hashabstrastarray_seed
```

Comment:
```suggestion
    h += hashabstractarray_seed
```
(Congrats, you got it wrong in both occurrences! :-p)

Reply:
The miracle of tab-completion!

Reply:
I was rather fond of that typo.